airbyte-integrations/bases/base-python/base_python/sdk/streams/auth/token.py

```diff
@@ -0,0 +1,11 @@
+from typing import Mapping, Any
+
+from base_python.sdk.streams.auth.core import HttpAuthenticator
+
+
+class TokenAuthenticator(HttpAuthenticator):
```

Comment:
We should call this one `BearerAuthenticator`

Reply:
the `Bearer` header is used also in OAuth and JWT so I think(?) it's not a correct name? some options: 

- leave as is
- `ApiTokenAuthenticator`
- `StaticTokenAuthenticator`
???

Reply:
The class name is using the type of the secret but doesn't say anything about authorization method used. You have no guarantee that the service is going to use `Bearer` as a method. 

How is JWT going to be different from this one?

Reply:
That's a good point. There's three separate things here: 
1. name of the header
2. auth method (bearer, basic etc..) 
3. actual value of the token (this can be obtained differently static 

maybe a better division is to have a single `Authenticator` class which takes the params: header name, auth method, and a `TokenManager` object which handles getting the token (JWT, OAuth, Static)

Reply:
On second thought I think I'm going to hold off on baking that into the `Authenticator` abstraction and will instead make it available in the implementing classes for now. I don't feel confident enough in having enough experience with all auth methods in the wild. See latest commit for this change.